easybuild/framework/easyblock.py

```diff
@@ -62,7 +62,7 @@
 from easybuild.framework.easyconfig.style import MAX_LINE_LENGTH
 from easybuild.framework.easyconfig.tools import get_paths_for
 from easybuild.framework.easyconfig.templates import TEMPLATE_NAMES_EASYBLOCK_RUN_STEP, template_constant_dict
-from easybuild.tools import config, filetools
+from easybuild.tools import config, filetools, run
```

Comment:
There's another line that needs to be changed at https://github.com/easybuilders/easybuild-framework/blob/9fd2b836ae9df5a5bff3f1bc926b115818b297fd/easybuild/framework/easyblock.py#L3047
and then you can remove `filetools` from this import.

Reply:
Done